tcms/utils/permissions.py

```diff
@@ -35,7 +10,7 @@ def initiate_user_with_default_setups(user):
     created user.
     """
     # create default permissions if not already set
-    assign_default_group_permissions()
+    Command.assign_default_group_permissions(assign_if_none=True)
```

Comment:
See comment above. 

- if you are passing the value to this argument then you don't need to have a default value in the method signature
- if you are not having a default value then the naming of this argument is different from the actual behavior. In fact naming with both the current implementation and with the one I suggested is suboptimal. Names must reflect what the behavior is.